pgctl/cli.py

```diff
@@ -78,6 +79,49 @@ def get_state(status):
     return str(state)
 
 
+def stat_service(status):
+    r"""
+    Parse a single line of svstat output.
+
+    >>> stat_service("date: up (pid 1202562) 1 seconds\n")
+    'date'
+
+    >>> stat_service("playground/date: down 0 seconds, normally up")
+    'playground/date'
+
+    """
+    return str(status.split(':')[0])
+
+
+def stat_pid(status):
+    r"""
+    Parse a single line of svstat output.
+
+    >>> stat_pid("date: up (pid 1202562) 1 seconds\n")
+    '1202562'
+
+    >>> stat_pid("playground/date: down 0 seconds, normally up")
+    ''
+
+    """
+    status = status.rstrip()
+    if 'pid' not in status:
+        return str('')
```

Comment:
`return str('')` is an odd way to return empty string :p

wonder if it would make more sense for this function to return an integer / None instead of a string / empty string?